src/main/java/com/oltpbenchmark/benchmarks/featurebench/FeatureBenchLoader.java

```diff
@@ -182,16 +182,26 @@ public void load(Connection conn) throws SQLException {
                 int batchSize = workConf.getBatchSize();
                 StringBuilder columnString = new StringBuilder();
                 StringBuilder valueString = new StringBuilder();
-
-                for (Map<String, Object> columnsDetails : this.columns) {
+                for (int index=0; index < this.columns.size(); index++) {
+                    Map<String, Object> columnsDetails = this.columns.get(index);
                     if (columnsDetails.containsKey("count")) {
                         for (int i = 0; i < (int) columnsDetails.get("count"); i++) {
                             columnString.append(columnsDetails.get("name") + String.valueOf(i + 1)).append(",");
-                            valueString.append("?,");
+                            if (this.baseutils.get(index).getInstance().getClass().getName().
+                                toLowerCase().indexOf("json") >= 0)
```

Comment:
we're not counting load phase towards the results, but this reflection use will definitely have performance impact(time taken to load the data).